src/context.ts

```
@@ -155,42 +202,70 @@ export default {
+      )
       layouts.push(`<template v-else>${defaultSlotSource}</template>`)
 
       return layouts.join('\n')
     }
     else {
       const props: string[] = [...nodeProps]
-      if (staticLayoutNameBind)
-        props.push(`is="layout-${staticLayoutNameBind.value?.content}-uni"`)
-      else
-        props.push(`:is="\`layout-\${${(dynamicLayoutNameBind.exp as SimpleExpressionNode).content}}-uni\`"`)
+      if (staticLayoutNameBind) { props.push(`is="layout-${staticLayoutNameBind.value?.content}-uni"`) }
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)